tests/cli/test_integration_datasets.py

```diff
@@ -269,8 +269,26 @@ def test_dataset_reimport_removed_dataset(runner, project, sleep_after):
 
 
 @pytest.mark.integration
+def test_dataset_import_preserve_names(runner, project, sleep_after):
+    """Test import keeps original file names."""
+    DOI = '10.7910/DVN/F4NUMR'
```

Comment:
Let's not define constants in local scopes, otherwise pylint is crying. Just make it normal variable (lower case).